app/javascript/mastodon/service_worker/entry.js

```diff
@@ -1 +1,10 @@
 import './web_push_notifications';
+
+// Cause a new version of a registered Service Worker to replace an existing one
+// that is already installed, and replace the currently active worker on open pages.
+self.addEventListener('install', function(event) {
```

Comment:
Yes `skipWaiting` should be fine in this case. Since we're not caching, there's no situation I can think of where the "frontend" would get out of sync with the "backend" (service worker).